hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/protocol/StorageContainerLocationProtocol.java

```diff
@@ -286,6 +286,23 @@ Pipeline createReplicationPipeline(HddsProtos.ReplicationType type,
    */
   boolean getReplicationManagerStatus() throws IOException;
 
+  /**
+   * Start ContainerBalancer.
+   */
+  void startContainerBalancer() throws IOException;
```

Comment:
According to our design decision, should we add the parameters like `threshold` and `consecutiveIdleIterations` in the start ?

Reply:
@GlenGeng Thanks for the review!  will add these and some usage info.

Reply:
updated. PTAL

Reply:
Seems these params are not passed from these RPCs in StorageContainerLocationProtocol ?

Reply:
yes , i keep it like this , because for now we can not determine how these parameters will be used or should we add more parameters(like bandwidth, which determines the maximum speed at which a block will be moved from one datanode to another ). I just implement the simplest function for now, and i think it will not be hard to complete the full implementation of this after we review the design doc.